libbeat/keystore/file_keystore.go

```diff
@@ -415,6 +415,11 @@ func (k *FileKeystore) Package() ([]byte, error) {
 	return k.loadRaw()
 }
 
+// ConfiguredPath returns the path to the keystore.
+func (k *FileKeystore) ConfiguredPath() string {
+	return k.Path
+}
+
```

Comment:
Yes. We can force it. But I think we need to add a warning about the config overwrites in Functionbeat. It might be confusing for users when they are looking for their keystore in the configured path and they cannot find it.

Reply:
I've seen it's already printed to the logs, so I haven't added anything. Just a new overwrite for keystore path.